docs/formats/zst.md

```diff
@@ -190,128 +117,132 @@ independently of each other, similar to zng compression framing.
+Zed values.
 
 #### Segment Maps
 
-The foundation of column reconstruction is based on segment maps, which is simply
-a list of the segments from the data area that are concatenated to form the
-data for a column stream.
+The foundation of column reconstruction is based on _segment maps_.
+A segment map is a list of the segments from the data area that are
+concatenated to form the data for a column stream.
 
 Each segment map that appears within the reassembly records is represented
-with a zng array of records that represent seek ranges, i.e., specifically
-this zng type:
+with a Zed array of records that represent seek ranges conforming to this
+type signature:
 ```
-array[record[offset:uint64,length:uint32]]
+[{offset:uint64,length:uint32}]
 ```
 In the rest this document, we will refer to this type as `<segmap>` for
 shorthand and refer to the concept as a "segmap".
 
 > We use the type name "segmap" to emphasize that this information represents
-> a set of seek ranges where data stored and must be read from *rather than*
+> a set of byte ranges where data stored and must be read from *rather than*
 > the data itself.
 
-#### The Root Reassembly Record
+#### The Super Column
 
-The first of the N+1 reassembly records defines the "root column", where this column
-represents the sequence of schemas of each original zng record, i.e., indicating
-which schema's column stream to select from to pull column values to form the zng row.
-The sequence of schemas is defined by each row's small-integer positional index,
-0 to N-1, within the N schemas.
+The first of the N+1 reassembly records defines the "super column", where this column
+represents the sequence of super types of each original Zed value, i.e., indicating
+which super type's column stream to select from to pull column values to form
+the reconstructed value.
+The sequence of super types is defined by each type's super ID (as defined above),
+0 to N-1, within set of N super types.
 
-The root column stream is encoded as zng int32 values.
-While there are a large number entries in the root column (one for each original row),
-the cardinality of the set of root identifiers is small in practice so this column
+The super column stream is encoded as a sequence of ZNG-encoded int32 primitive values.
+While there are a large number entries in the super column (one for each original row),
+the cardinality of super IDs is small in practice so this column
 will compress very significantly, e.g., in the special case that all the
-rows have the same schema, this column will compress to practically nothing.
+values in the ZST object have the same super ID,
+the super column will compress trivially.
 
-The type of the reassembly record for the root column stream is simply:
+The type of the reassembly record for the super column stream has the
+type signature:
 ```
-record[root:<segmap>]
+{root:<segmap>}
 ```
 
-#### The Row Reassembly Records
+#### The Reassembly Records
```

Comment:
I would consider dropping the "The" here.
```suggestion
#### Reassembly Records
```

Reply:
There are a specific number of reassembly records in the file and this section is discussing the particular layout of these records.  "Reassembly Records" would be a good title for a section that generically described what reassembly records are so by removing "The" you are changing the meaning.